packages/i18n/src/translator.ts

```diff
@@ -1,8 +1,9 @@
-/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable camelcase */
 import Mustache from 'mustache';
-import Catalog from './catalog';
+import type Catalog from './catalog';
```

Comment:
had no idea this was possible

Reply:
yeah, it's a relatively recent addition but can be really nice, in particular because the modules are not actually loaded this way